testsuite/testcases/src/lib.rs

```diff
@@ -104,6 +104,8 @@ pub enum LoadDestination {
     AllNodes,
     AllValidators,
     AllFullnodes,
+    // Send to AllFullnodes, if any exist, otherwise to AllValidators
+    AllFullnodesIfAnyElseAllValidators,
```

Comment:
nit: this is very long 😄  Maybe `FullnodesOtherwiseValidators` with a comment?

Reply:
was going back and forth on what is better. since this is really rarely used in code, I've decided to keep it long. we can change it to short if it ends up being used in multiple places